pydash/pydash_app/dashboard/aggregator/statistics.py

```diff
@@ -76,8 +85,13 @@ def field_name(self):
     def append(self, endpoint_call, dependencies):
         self.value += endpoint_call.execution_time
 
+    def rendered_value(self):
```

Comment:
Couldn't this be moved to `FloatStatisticABC`? I don't have much experience with abstract base classes in Python though.

Reply:
You're entirely right on this point (and the others). I guess I forgot to move it up there once I decided to have `FloatStatisticABC`s render only a single floating point value. :+1: